### DISCUSSION TITLE
Docs page is missing?

### DISCUSSION THREAD

**Discussion by PiousVenom** [spatie/laravel-permission#1751] [Q&A]
I was looking at the docs for this just yesterday, but today they seem to be missing. The link I'm following is:

https://spatie.be/docs/laravel-permission/v4/introduction

Is this being discontinued?

**Comment by drbyte** [comment]
Seems there's something wrong with the docs server

In the meantime, you can see all the docs in the [/docs dir](https://github.com/spatie/laravel-permission/tree/master/docs) of this repo.

/cc @riasvdv 
// ref for Rias ... this is how https://spatie.be/docs/ looks right now:
![Documentation  Spatie 2021-06-01 at 11 54 59 PM](https://user-images.githubusercontent.com/404472/120421762-e30e1a00-c334-11eb-9536-6bf0ec50d146.jpg)
and
<img width="627" alt="Screen Shot 2021-06-01 at 11 57 01 PM" src="https://user-images.githubusercontent.com/404472/120421876-1e104d80-c335-11eb-917c-153b7b52a7ae.png">

**Reply by PiousVenom** [reply]
Thanks. Been trying to figure out how to "give" non-logged in page visitors permissions to view specific items, like the `Login` button.

**Reply by drbyte** [reply]
Not sure why you'd need to "give" a not-logged-in user permission to see a "login" button.
That wouldn't be user-permission-related, because you don't know who the user is, so there's no "permission" to "give".

Perhaps you're looking for something within Laravel about whether the user "is" logged-in or not. That's completely outside of what this package offers, since it's built-in to Laravel if you've installed basic auth stuff in Laravel using something like Jetstream or laravel-ui or one of those.
